httpmw/requestid/requestid.go

```diff
@@ -0,0 +1,77 @@
+package requestid
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	CorrelationIDKey      = "correlationID"
+	ARMClientRequestIDKey = "armClientRequestID"
+	ClientSessionIDKey    = "clientSessionID"
+
+	// Details can be found here:
+	// https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#client-request-headers
+	RequestCorrelationIDHeader = "x-ms-correlation-request-id"
+	// RequestARMClientRequestIDHeader  Caller-specified value identifying the request, in the form of a GUID
+	RequestARMClientRequestIDHeader = "x-ms-client-request-id"
+	// RequestClientSessionIDHeader is the http header name ARM adds for the client session id. AKS treats it as operation
```

Comment:
Add URL. I cannot find the doc about this.

Reply:
Replacing clientsessionid header with acsoperationid header as one of the defaults